src/components/Hero/HubHero/index.tsx

```diff
@@ -38,55 +39,51 @@ const HubHero = ({
         }}
       />
       <Stack
-        spacing="4"
-        p={{ base: "4", lg: "8" }}
-        textAlign={{ base: "center", xl: "start" }}
-        borderRadius={{ xl: "base" }}
-        bg={{ xl: "hubHeroContentBg" }}
-        position={{ xl: "absolute" }}
-        maxW={{ xl: "sm" }}
-        top={{ xl: "50%" }}
-        transform={{ xl: "translateY(-50%)" }}
-        backdropFilter={{ xl: "auto" }}
-        backdropBlur={{ xl: "base" }}
-        wordBreak="break-word"
-        sx={{
-          "inset-inline-start": { xl: "32px" },
-        }}
+        className={cn(
+          "gap-4 p-4 lg:p-8",
+          "text-center xl:text-start",
+          "xl:rounded",
+          "xl:bg-hub-hero-content-bg",
+          "xl:absolute xl:top-1/2 xl:max-w-sm",
+          "xl:-translate-y-1/2 xl:transform",
+          "xl:backdrop-blur xl:backdrop-filter",
+          "break-words",
+          "xl:start-[32px]"
+        )}
       >
         {title ? (
-          <Text
-            as="h1"
-            size="md"
-            color="body.medium"
-            fontWeight="normal"
-            textTransform="uppercase"
-          >
+          <h1 className="text-md font-normal uppercase text-body-medium">
             {title}
-          </Text>
+          </h1>
         ) : null}
         <Stack
-          alignSelf="center"
-          spacing={{ base: "2", md: "1" }}
-          maxW="container.md"
+          // --eth-sizes-container-md: 768px ==> tailwind max-w-3xl
+          className="max-w-3xl gap-2 self-center md:gap-1"
```

Comment:
```suggestion
          className="max-w-screen-md gap-2 self-center md:gap-1"
```

Reply:
As a workaround it works, but for 
`container-sm` should be `640px` while tailwind `w-screen-sm`  has size of `480px`

Reply:
Oh, I get it. Ok, so this is what happens: chakra has two separate values for containers and breakpoints, while tw uses the same value for both.

So from now on, we will no longer have the `640px` value in our theme. This means that in cases where we are using `container.sm` to get the 640px, we will have to rethink why we need that specific size and refactor accordingly.

In this case, since we are setting the container size, `max-w-screen-md` is well used and makes sense because we don't want this box to be bigger than this breakpoint size.